### ISSUE TITLE
Disable mode indicator

### ISSUE THREAD

**Issue by alvoro** [denesb/xfce4-i3-workspaces-plugin#53]
Hello. Is there a way to hide mode indicator? I use another bar for it. Thanks

**Comment by denesb** [comment]
No, there isn't. Feel free to add a configuration item for it and open a PR.
